### DISCUSSION TITLE
LDAP / Active Directory integration

### DISCUSSION THREAD

**Discussion by MB926** [provectus/kafka-ui#4354] [Q&A]
Hi Folks,

Is there any who can configure or know that  LDAP / AD integration with KAFKA-UI. Below is my values.yaml 

env:
  - name: LOGGING_LEVEL_ROOT
    value: info
  - name: LOGGING_LEVEL_COM_PROVECTUS
    value: info 
  - name: KAFKA_CLUSTERS_0_NAME
    value: Kafka_Cluster
  - name: KAFKA_CLUSTERS_0_BOOTSTRAPSERVERS
    value: xxxx:9092
  - name: KAFKA_CLUSTERS_1_NAME
    value: Dev_Kafka_Cluster
  - name: KAFKA_CLUSTERS_1_BOOTSTRAPSERVERS
    value: xxxx:9092
  - name: KAFKA_CLUSTERS_1_READONLY
    value: "true"
  - name: KAFKA_CLUSTERS_0_READONLY
    value: "true" 

I read the kafka UI docs and below is LDAP guide but I don't know how or where I put that config on values.yaml

auth:
  type: LDAP
spring:
  ldap:
    urls: ldap://localhost:10389
    base: "cn={0},ou=people,dc=planetexpress,dc=com"
    admin-user: "cn=admin,dc=planetexpress,dc=com"
    admin-password: "GoodNewsEveryone"
    user-filter-search-base: "dc=planetexpress,dc=com"
    user-filter-search-filter: "(&(uid={0})(objectClass=inetOrgPerson))"
    group-filter-search-base: "ou=people,dc=planetexpress,dc=com" # required for RBAC
oauth2:
  ldap:
    activeDirectory: false
    aсtiveDirectory:
      domain: memelord.lol

**Comment by waitspring** [comment]
1. use vim make a kafka-ui configure file like `/etc/kafka-ui/application.yml`
2. run kafka-ui like `java -Dspring.config.additional.location=/etc/kafka-ui/application.yml -jar kafka-ui-api-v0.7.0.kar &`
